build.sbt

```diff
@@ -3,7 +3,7 @@ import PgpKeys._
 cancelable in Global := true
 
 inThisBuild(Seq(
-  scalaVersion := "2.12.3",
+  scalaVersion := "2.11.11",
```

Comment:
please don't

Reply:
i did that for local publishing, ill clean those kind of things later (version too).

Reply:
For publishing you can also do `++2.11.11` in SBT